### PR TITLE
[vsg] Update to version 1.1.7

### PR DESCRIPTION
**Pull request by gwaldron**
Updates the vsg (VulkanSceneGraph) port to version 1.1.7.

Removed the patch `devendor-glslang.patch` because its functionality was integrated into the core package as of version 1.1.5.
(See: https://github.com/vsg-dev/VulkanSceneGraph/pull/1216/files)

- [x] Changes comply with the [maintainer guide](https://github.com/microsoft/vcpkg-docs/blob/main/vcpkg/contributing/maintainer-guide.md).
- [x] SHA512s are updated for each updated download.
- [x] The "supports" clause reflects platforms that may be fixed by this new version.
- [x] Any fixed [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt) entries are removed from that file.
- [x] Any patches that are no longer applied are deleted from the port's directory.
- [x] The version database is fixed by rerunning `./vcpkg x-add-version --all` and committing the result.
- [x] Only one version is added to each modified port's versions file.

(edited: added reason for removing patch)

### CODE REVIEW

**Comment by gwaldron** [comment]
@microsoft-github-policy-service agree

Glenn Waldron

File: ports/vsg/devendor-glslang.patch
Comment (WangWeiLin-MV) [inline ports/vsg/devendor-glslang.patch:1]: Thanks for your updates. Would you mind adding the brief PR description for removing this patch?

Comment (gwaldron) [reply ports/vsg/devendor-glslang.patch:1]: Done!

**Review by WangWeiLin-MV — APPROVED** [review]
The port installation tests pass with the following triplets:

* x64-linux

**Review by data-queue — APPROVED** [review]

**Pull request merged**